projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,13 @@
+<cdk-virtual-scroll-viewport appendOnly itemSize="80" class="tabs-viewport">
```

Comment:
When you click apply, the table jumps to another position. Do you know why?

Reply:
I couldn't figure out why it does that, but stopped it from jumping by remembering the scroll position and reapplying it later in [720a718](https://github.com/geo-engine/geoengine-ui/pull/614/commits/720a718fd695adfa16c1884236e53a86b7901094)